### DISCUSSION TITLE
Ignore vanished players for sle

### DISCUSSION THREAD

**Discussion by ChargerSRT64** [EssentialsX/Essentials#4087] [Q&A] [answered]
One player sleep without the sleeping ignored? Also this feature doenst seem to work with supervanish or vanishnopacket in that if you vanish and unvanish one player sleep breaks and you have to be in vanish. If you join unvanished it works until you vanish and then unvanish in both plugins. for one player sleep to work all players who have vanish perms have to be vanished for other vanish plugins. it works ok for essentials vanish.

**Comment by JRoy** [comment, marked as answer]
This is not in the scope of Essentials
